src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -77,10 +85,21 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
         return paymentMethod === ACH || paymentMethod === CREDIT_CARD;
     }
 
+    tokenizeEventAction() {
+        return this._currentPaymentMethod === ACH
+            ? TOKENIZE_ACH_EVENT_ACTION
+            : TOKENIZE_CREDIT_CARD_EVENT_ACTION;
+    }
+
     requestSetPaymentMethod(paymentMethod) {
         this.isLoading = true;
-        const iframe = this.template.querySelector(`[data-id='${this.CUSTOM_LABELS.commonPaymentServices}']`);
-        tokenHandler.setPaymentMethod(iframe, paymentMethod, this.handleError, this.resolveSetPaymentMethod);
+        const iframe = this.template.querySelector(
```

Comment:
Looks like this is done in a few places.  What do you think about putting this iframe selector into a class level constant/function so it can be reused?